Refactor unclear wording re vulnerabilities (#90)
remarkjs/remark-math: readme.md

## Security

Assuming you trust KaTeX/MathJax, using `rehype-katex`/`rehype-mathjax` is

> safe. If a vulnerability were to be introduced in them, it could open you up to
> a [cross-site scripting (XSS)][wiki-xss] attack. See their readmes for more
> info.

Reviewer: ```suggestion
safe.
If a vulnerability is introduced in them, it opens you up to a
[cross-site scripting (XSS)][wiki-xss] attack.
See their readmes for more info.
```

Formatting. Also uses active language instead of verbose “were to be” and “could”
